packages/core/src/sessionInfo/ISessionInfoManager.ts

```diff
@@ -39,7 +39,9 @@ export interface ISessionInfoManagerOptions {
  */
 export interface ISessionInfoManager {
   update(sessionId: string, options: ISessionInfoManagerOptions): Promise<void>;
-  get(sessionId: string): Promise<ISessionInfo | undefined>;
-  getAll(): Promise<ISessionInfo[]>;
+  get(
```

Comment:
I'm not 100% on how/if this is exposed to the developer, but does it not potentially make more sense to have separate getters for `ISessionInternalInfo`?

Reply:
It isn't exposed to developers at all, this is part of the purely internal API. Basically, the only difference between the two functions would be the set of keys they access in memory, but I don't think I would add a separate function for that. One change that could be possible is to have the type `ISessionInternalInfo` include the type `ISessionInfo`, and only use the latter as part of the public API. We could then have a `getPublic` function to get `ISessionInfo`, and `getinternal` to get all the values in memory. What do you think ?

Reply:
Effectively we already only use `ISessionInfo` as part of the public API, right (and the union type only in internal API's). So in that case I think no changes are necessary.